components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -859,6 +894,12 @@ void RRTMGPRadiation::run_impl (const double dt) {
         d_cldmed(icol) = cldmed(i+1);
         d_cldhgh(icol) = cldhgh(i+1);
         d_cldtot(icol) = cldtot(i+1);
+        d_T_mid_at_cldtop(icol) = T_mid_at_cldtop(i+1);
+        d_p_mid_at_cldtop(icol) = p_mid_at_cldtop(i+1);
+        d_cldfrac_ice_at_cldtop(icol) = cldfrac_ice_at_cldtop(i+1);
+        d_cldfrac_liq_at_cldtop(icol) = cldfrac_liq_at_cldtop(i+1);
```

Comment:
These are all 1d views, so there's no C vs F90 ordering. I assume YAKL does offer unmanaged arrays, so one could simply create `cldfrac_XYZ_at_cldtop` using the same data ptr from the kokkos views, and skip this copy after the calculation...

Reply:
Done. #2466

Reply:
Sounds good.

Reply:
YAKL does offer unmanaged arrays/views which can use the kokkos data directly.